js/queen.js

```diff
@@ -0,0 +1,126 @@
+import { Piece } from "./piece.js"
+
+/*
+* Represents a queen chess piece that is used with the Board class
+*/
+export class Queen extends Piece{
+    /**
+     * Instantiates a queen piece with the given position and piece ID.
+     * The piece belongs to the player with the given player ID.
+     * 
+     * @param {int} id Piece's ID.
+     * @param {int} rowPos Piece's row position.
+     * @param {int} colPos Piece's column position.
+     * @param {int} playerOwner
+     * 
+     */
+    constructor(id, rowPos, colPos, playerOwner) {
+        super();
+        this.id = id;
+        this.rowPos = rowPos;
+        this.colPos = colPos;
+        this.playerOwner = playerOwner;
+    }
```

Comment:
Don't forget to set the `unicodeChar` attribute for the `Queen` class in the constructor.

Reply:
Added unicode.